docs/content/plugins/writing-a-vendure-plugin.md

```diff
@@ -141,7 +141,7 @@ export class RandomCatPlugin {}
 
 ### Step 6: Declare any providers used in the resolver
 
-In order that out resolver is able to use Nest's dependency injection to inject and instance of `CatFetcher`, we must add it to the `providers` array in our plugin:
+In order that out resolver is able to use Nest's dependency injection to inject an instance of `CatFetcher`, we must add it to the `providers` array in our plugin:
```

Comment:
ah there are actually 2 typos! "In order that **out** resolver" is the one I thought you were talking about. Looks like I was typing too fast on that day 😆

Reply:
Done! :)